app/uk/gov/nationalarchives/omega/editorial/views/editSetRecordEdit.scala.html

```diff
@@ -171,7 +171,7 @@
             val creatorItems = creators.map(creator =>
                 SelectItem(
                     selected = currentId.contains(creator.identifier),
-                    text = creator.description.displayedName,
+                    text = creator.description.head.displayedName,
```

Comment:
I have now modified `AgentSummary` so that the `AgentDescription`s are in a `cats.data.NonEmptyList`, rather than a `List`. I'm assuming that there would always be at least one description as without a description there will be no label that we can display in the frontend. The behaviour of the UI if there is more than one description has not been specified.